homeassistant/components/notify/legacy.py

```diff
@@ -120,6 +114,12 @@ async def async_platform_discovered(
         hass, DOMAIN, async_platform_discovered
     )
 
+    return [
+        async_setup_platform(integration_name, p_config)
```

Comment:
If we want to use `asyncio.wait` for these we need to create tasks. Passing coroutines directly to wait is deprecated.

https://docs.python.org/3/library/asyncio-task.html#asyncio.wait

Reply:
Good catch, will rewrap them

Reply:
e74146178b0f00c64cf2e2718497df9e887afd8c